pkg/remoteconfig/client/products/apmsampling/remote_rates.go

```diff
@@ -14,6 +14,11 @@ package apmsampling
 // This list is not exhaustive.
 type SamplingMechanism uint32
 
+const (
+	//todo(shevchenko): move or add other constants
+	SpanSamplingRuleMechanism SamplingMechanism = 8
```

Comment:
```suggestion
	SamplingMechanismSpan SamplingMechanism = 8
```
Or `SamplingMechanismSingleSpan` if you prefer, to use the type prefix